tests/foreman/api/test_subscription.py

```diff
@@ -108,19 +111,14 @@ def test_positive_refresh(request):
 
     :CaseImportance: Critical
     """
-    org = entities.Organization().create()
+    org = function_entitlement_manifest_org
     sub = entities.Subscription(organization=org)
-    with manifests.original_manifest() as manifest:
-        upload_manifest(org.id, manifest.content)
```

Comment:
are we good with replacing `manifests.clone()` and `manifests.original_manifest()` both with entitlement_manifest_org?

Reply:
Yes, this should cause no issues and should be more flexible than both `manifests.clone()` and `manifests.original_manifest()`.